_posts/2020-05-13-cockpit-219.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 219
+author: kkoukiou
+date: 2020-05-13
+tags: cockpit linux
+slug: cockpit-219
+category: release
+summary: Cockpit with XYZ
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 219.
+
+### Logs: Improved journal logs filtering
+
+It's now possible to filter logs by providing keywords in a text filter which may include the unit of interest, time constraints, priority and [journal fields].
+There is also the possibility to pause/resume the logs streaming.
+Lastly journal logs which appear in other pages are now linked to the the full journal in the Logs page.
+
+![Journal Logs Filtering](/images/journal-logs-filtering.png)
+![Journal Logs Per Page Link](/images/journal-logs-per-page-link.png)
+
+### Administative access improvements
+
+Cockpit now allows more control over your access level. You can gain
+administrative access in a running session, and Cockpit will remember
+that for the next session.
```

Comment:
I'd like to mention this, as that's the real killer feature:

... This finally allows the user to do administrative actions without a password on the login page, such as when using smart card or Kerberos authentication, or ssh/Dashboard login with an SSH public key.